components/Card.js

```diff
@@ -93,13 +93,13 @@ CardAddress.propTypes = {
   label: PropTypes.string,
 };
 
-export function PositionRequirements({ label, children }) {
+export function PositionRequirements({ label, requirements }) {
```

Comment:
I replaced the `children` nodes prop with the `requirements` array prop. Please let me know if you don't want this and I can either update this PR or close it. 😅